src/commands.rs

```diff
@@ -61,6 +61,9 @@ pub fn explain(code: &CheckCode, format: SerializationFormat) -> Result<()> {
                 })?
             );
         }
+        SerializationFormat::Junit => {
+            todo!()
```

Comment:
I don't think it's useful to have `--explain` to output junit xml.

Reply:
I agree, log some feedback?